util/mod_translation_updater.py

```diff
@@ -463,6 +463,8 @@ def update_tr_file(dNew, mod_name, tr_file):
 
 # Updates translation files for the mod in the given folder
 def update_mod(folder):
+	if not os.path.exists(os.path.join(folder, "init.lua")):
+		print(f"Warning: Mod folder {folder} is missing init.lua.")
```

Comment:
`return` below also avoids that `locale` directories are created when the script is run from the game's root dir.

Reply:
I think I'll `exit(1)`, even, which seems to have been the intention of the code for folders which do not belong to mods. (`get_modname` was supposed to return `None` for modpacks instead of defaulting to the folder name, but the check got out of date (checks for `modpack.txt`) hence it didn't work as expected.)